docs/upgrade.adoc

```diff
@@ -5,6 +5,54 @@
 Please follow these instructions when upgrading from Titan or an older
 JanusGraph release.
 
+=== Upgrading to JanusGraph 0.3.0
+*IMPORTANT* You should back-up your data prior to attempting an upgrade! Also please note that once an upgrade has been completed you will no longer be able to connect to your graph with client versions prior to 0.3.0.
```

Comment:
Please change this `*IMPORTANT*` to `IMPORTANT:` so that it creates a call out box.

Reply:
It has been changed to ```IMPORTANT:```